src/status_im2/contexts/quo_preview/standart_auth/keycard_pin_entry.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im2.contexts.quo-preview.standart-auth.keycard-pin-entry
+  (:require [quo2.core :as quo]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [utils.re-frame :as rf]
+            [status-im2.common.standard-auth.keycard.view :as standard-auth]))
+
+(def descriptor
```

Comment:
In this descriptor we should ideally have the `max-digits` field (and as I commented below, the `theme` field has no effect). The goal is for the person reviewing the component to be able to see all variations without having to change the code and rebuild.